app/routes/comment.js

```diff
@@ -3,37 +3,43 @@ const router = express.Router()
 const auth = require('../middleware/auth')
 const commentController = require('../controllers/comment')
 
+// CREATE COMMENT
 router.post(
-  '/',
+  '/:id',
```

Comment:
@Rupeshiya how can you get the ID before even creating the comment

Reply:
@devesh-verma It is a POST ID on which comment is going to be added!